bundle/manifests/node-observability-operator.clusterserviceversion.yaml

```diff
@@ -434,6 +434,76 @@ spec:
           verbs:
           - create
           - patch
+        - apiGroups:
```

Comment:
We don't need these files anymore then:

- https://github.com/openshift/node-observability-operator/blob/bf1a0755825a1da11dc2a50238f024c23eb098b9/bundle/manifests/node-observability-operator_rbac.authorization.k8s.io_v1_role.yaml
- https://github.com/openshift/node-observability-operator/blob/bf1a0755825a1da11dc2a50238f024c23eb098b9/bundle/manifests/node-observability-operator_rbac.authorization.k8s.io_v1_rolebinding.yaml